components/Terminal/index.vue

```diff
@@ -40,9 +44,25 @@ function handleCommand(command: string): string | undefined {
 function handleInput(event: KeyboardEvent) {
   const { key } = event;
 
-  if (key === "Tab") return;
   event.preventDefault();
 
+  if (key === "Tab") {
```

Comment:
N.b. Tab is used by default in browsers for keyboard navigation, perhaps its worth providing an escape method? (Shift+Tab or something)

Reply:
how about esc, which we could use to set focus to the next element after the terminal. if we set in front of it everything after it becomes inaccessible

Reply:
That would work.  When we do the initial message, we probably want to alert the user to use escape to exit, but that's not difficult

Reply:
when someone is pressing tab or when someone is pressing esc?

Reply:
I was thinking of doing some kind of welcome message when the terminal starts. Something like this (but probably shorter):
![20230823_12h13m13s_grim](https://github.com/lancasterunicompsoc/CompSoc-Website/assets/41267740/7a47d841-0066-439f-97d3-6fb22f74899d)